salt/core/mail.sls

```diff
@@ -0,0 +1,51 @@
+# Install and configure email software
+postfix:
+  pkg:
+    - installed
+  service:
+    - running
```

Comment:
Elsewhere we use `service.running`. Should we use it here for consistency?

Other services also `enable: True`. Do we not want postfix to be enabled at boot time?

Reply:
I've updated this to use `service.running` and added `enable: True`.